plugins/chat/assets/javascripts/discourse/controllers/chat-channel-edit-description.js

```diff
@@ -2,20 +2,29 @@ import Controller from "@ember/controller";
 import { action, computed } from "@ember/object";
 import ModalFunctionality from "discourse/mixins/modal-functionality";
 import ChatApi from "discourse/plugins/chat/discourse/lib/chat-api";
+import { tracked } from "@glimmer/tracking";
 
 export default class ChatChannelEditDescriptionController extends Controller.extend(
   ModalFunctionality
 ) {
-  editedDescription = "";
+  @tracked editedDescription = "";
+  DESCRIPTION_MAX_LENGTH = 280;
```

Comment:
I prefer to have constants outside of the class

Reply:
Ah I like the idea of that. Just pushed an update :)